larky/src/main/java/com/verygood/security/larky/objects/LarkyPyObject.java

```diff
@@ -0,0 +1,165 @@
+package com.verygood.security.larky.objects;
+
+import com.google.common.collect.ImmutableCollection;
+import com.google.common.collect.ImmutableSortedSet;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Objects;
+
+import com.verygood.security.larky.objects.type.LarkyType;
+import com.verygood.security.larky.parser.StarlarkUtil;
+
+import net.starlark.java.eval.Dict;
+import net.starlark.java.eval.EvalException;
+import net.starlark.java.eval.Printer;
+import net.starlark.java.eval.Starlark;
+import net.starlark.java.eval.StarlarkThread;
+import net.starlark.java.eval.Tuple;
+
+import org.jetbrains.annotations.NotNull;
+
+
+public class LarkyPyObject implements PyObject, Comparable<LarkyPyObject> {
```

Comment:
Can you please provide a brief about differences between:
- LarkyPyObject
- LarkyTypeObject

And why first one can't be instantiated thru `__new__`